### DISCUSSION TITLE
Low Level example not working

### DISCUSSION THREAD

**Discussion by jamesguan** [NicoHood/PinChangeInterrupt#49] [Q&A]
Hello,

I'm trying to use the low level example, however, I'm not able to get it to compile. The lowlevel example with callbacks would be very useful for my use case. Is anyone else compile and run the example for the mega2560? What am I doing wrong?

https://github.com/NicoHood/PinChangeInterrupt/blob/master/examples/PinChangeInterrupt_LowLevel/PinChangeInterrupt_LowLevel.ino

**Comment by NicoHood** [comment]
What is the compile error?
